frontend/app/timed-admin/detail/controller.js

```diff
@@ -80,10 +82,30 @@ export default Controller.extend({
         )
       })
     } catch (e) {
-      this.notify.error(this.i18n.t('timed.reload.error-loading'))
+      this.notify.error(this.i18n.t('timed.detail.errorLoading'))
     }
   }).drop(),
 
+  fetchReports: task(function*() {
+    try {
+      const reports = yield this.store.query('timed-report', {
+        project: this.get('model'),
+        include: 'user',
+        ordering: '-date',
+        page: this.get('reportsPage'),
+        page_size: 20
+      })
+
+      reports.forEach(report => this.reports.pushObject(report))
+      this.set('reportsPage', this.get('reportsPage') + 1)
+      this.set('reportsNext', !!reports.get('links.next'))
```

Comment:
`Boolean(reports.get('links.next'))` would probably be a more readable way

Reply:
Changed